backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -98,6 +99,20 @@ module.exports = class CrmPage {
     await Promise.all([
       this.engine.click(selector),
       this.engine.waitForNavigation()
-    ]);      
+    ]);
+  }
+
+  /**
+   * Checks if there are no price fields present on the page or not
+   *
+   */
+  async checkIfPriceFieldsAreEmpty () {
+    // checks for the messages div - appears if the list is empty.
+    const priceFieldsEmpty = await this.engine.$('#crm-main-content-wrapper .messages');
+    if (priceFieldsEmpty) {
```

Comment:
```js
if (priceFieldsEmpty) {
  console.warn('No Price Field list present!');
  console.log('Taking the "Price Fields List" Page screenshot..'); // the whole point is not to dup the console.log everywhere
 
  return Promise.reject();
} else {
  return Promise.resolve();
}
```

You will also need to pass some argument since your console.log messages are slightly different.

So the function would be smth like:

```js
checkIfPriceFieldsAreEmpty (type) {
  // ...
  console.warn('No ' + type + ' list present!');
  console.log........
```

Reply:
Done!